translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -9510,11 +9687,10 @@ Value *SPIRVToLLVM::transGLSLExtInst(SPIRVExtInst *ExtInst,
         Val, VectorType::get(getBuilder()->getFloatTy(), 2));
     Value *Multiplier =
         getBuilder()->CreateFDiv(ConstantFP::get(Val->getType(), 1.0),
-                               ConstantFP::get(Val->getType(), 32767.0));
+                                 ConstantFP::get(Val->getType(), 32767.0));
```

Comment:
please remove unnecessary CreateFDiv in this comment

Reply:
I have resolved this in #168.